ios/keyman/Keyman/Keyman.xcodeproj/project.pbxproj

```diff
@@ -1080,12 +1106,11 @@
 				GCC_PREPROCESSOR_DEFINITIONS = "$(inherited)";
 				INFOPLIST_FILE = "Keyman-Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
-				KEYMAN_ROOT = "$(SRCROOT)/../../..";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
 				ONLY_ACTIVE_ARCH = NO;
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = "Tavultesoft.${PRODUCT_NAME:rfc1034identifier}";
+				PRODUCT_BUNDLE_IDENTIFIER = "Tavultesoft.${PRODUCT_NAME:rfc1034identifier}.dev";
```

Comment:
This is set for both `Debug` and `Debug + Sentry` configurations.  `Release` is unchanged.

Basically, this will help us distinguish errors from development versions (during our testing) from those in production, as Sentry will pass along the `.dev` tag from the development builds.

Reply:
Note that this scheme is replaced in the follow-up PR #2782, wherein we use Sentry's "Environment" setting instead, which is explicitly designed to assist with filtering.